src/main/UI/ComboBox.java

```diff
@@ -0,0 +1,24 @@
+package UI;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ComboBox {
+    JComboBox comboBox;
+
+    public ComboBox() {
+    }
+
+    public JComboBox getComboBox() {
+        return comboBox;
+    }
+
+    public void createComboBox(JFrame dialogBox, String[] comboBoxItems, Color foregroundcolor, int boundX, int boundY, int boundWidth, int boundHeight) {
```

Comment:
There's a Rectangle class that could encapsulate the bounds and size. That would cut down on the number of parameters and perhaps make this easier to read?